viime/models.py

```diff
@@ -537,30 +550,14 @@ class ModifyLabelListSchema(Schema):
     changes = fields.List(fields.Nested(ModifyLabelChangesSchema), required=True)
 
 
-listen(CSVFile, 'after_update', lambda mapper, connection, target: clear_cache(csv_file=target))
-listen(TableRow, 'after_update',
-       lambda mapper, connection, target: clear_cache(csv_file=target.csv_file))
-listen(TableColumn, 'after_update',
-       lambda mapper, connection, target: clear_cache(csv_file=target.csv_file))
-
-
 # Cached class method implementations:
 # These are placed here because dogpile does not allow you
 # to include the class instance in the cache key.  :'(
-@region.cache_on_arguments()
-def _read_csv_file(path: str, **kwargs):
-    return pandas.read_csv(path, **kwargs)
-
+# TODO: remove rest of cached class methods
```

Comment:
Might as well move these functions into the `CSVFile` class.  Also, `_headers` and `_keys` logic can be moved directly into the property methods.

Reply:
The `_headers` and `_keys` functions each take in parameter so they couldn't be moved into the property methods, so I just moved them into the`CSVFile` class as-is. Not sure if that is the ideal solution though.

Reply:
Okay, that works.